src/custom/utils/metadata.ts

```diff
@@ -1,3 +1,9 @@
+import CID from 'cids'
+import multihashes from 'multihashes'
+import pinataSDK from '@pinata/sdk'
+import safeStringify from 'fast-safe-stringify'
```

Comment:
Why `JSON.stringify` was not good enough? did we run into issues?

Reply:
No issues, I suggested it to make the serialization deterministic. No strong need though

Reply:
I was just curious